src/flask_multipass_keycloak.py

```diff
@@ -209,12 +208,16 @@ def _get_api_session(self):
         if api_token:
             api_session.headers.update({'Authorization': f'Bearer {api_token}'})
             return api_session
-        basic = HTTPBasicAuth(self.keycloak_settings['client_name'], self.keycloak_settings['client_secret'])
-        data = {'username': self.keycloak_settings['username'],
-                'password': self.keycloak_settings['password'],
-                'grant_type': 'password'}
+        data = {'client_id': self.keycloak_settings['client_id'],
```

Comment:
It seems like `client_id` was not required before for `password` grant type. Are we requiring it now?

Reply:
Well, `client_name` was used but in a weird way involving `HttpBasicAuth`. I ditched that approach just sending simple POST with form-encoded data. During that process I realized that key should be `client_id` instead of `client_name` so I changed it.